templates/beta-post.adoc

```diff
@@ -0,0 +1,194 @@
+//
+// If adding image into the post add :
+// -------------------------
+// [.img_border_light]
+// image::img/blog/FILE_NAME[IMAGE NAME/DESCRIPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not            // around diagrams. Then double check how it looks.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IAMGE NAME/DESCRIPTION" to a couple words of what the image is
+// // // // // // // //
+
+BETA_RELEASE_SUMMARY
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to a quick sentence intro. Make sure to keep the blank lines 
+// // // // // // // //
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a new, larger package that contains all Open Liberty beta and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try).
+
+If you try either package, <<feedback, let us know what you think>>.
+
+[#allbeta]
+== All Beta Features package
+
+As well as the Jakarta EE 9 features listed above, the new All Beta Features package contains our new FEATURES features which have been available in our link:/downloads/#development_builds[nightly builds] for a few weeks but are now easy to install from Maven Central too.
+// // // // // // // //
+// Above:
+// Change FEATURES to a feature that is included in this release     // version.
+// // // // // // // //
+
+[#SUB_TAG_1]
+=== SUB_FEATURE_TITLE
+
+// // // // // // // //
+// FURTHER EXPLENATION OF THE FEATURE/CHANGE ABOVE
+// // // // // // // //
+
+
+
+[source, java]
+----
+// // // // // // // //
+// EXAMPLE CODE
+// // // // // // // //
+----
+
+
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[source, xml]
+----
+
+----
+
+=== Try it now section for the feature 
+
+// // // // // // // //
+// how to try it now
+// // // // // // // //
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>20.0.0.RELEASE_VERSION-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.RELEASE_VERSION-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
```

Comment:
again I suppose you can keep this here for now, but the Jakarta EE 9 beta features is a point-in-time section of the beta blog post and once Jakarta EE 9 GAs, you'll need to remove it from the beta template